public/plugin.tsx

```diff
@@ -64,6 +77,82 @@ export class AlertingPlugin implements Plugin<AlertingSetup, AlertingStart, Aler
       },
     });
 
+    if (core.chrome.navGroup.getNavGroupEnabled()) {
+      // register applications with category and use case information
+      core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
+        {
+          id: PLUGIN_NAME,
+          category: DEFAULT_APP_CATEGORIES.detect,
+          showInAllNavGroup: true
+        }
+      ])
+      core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS['security-analytics'], [
+        {
+          id: PLUGIN_NAME,
+          category: DEFAULT_APP_CATEGORIES.detect,
+          showInAllNavGroup: false
+        }
+      ])
```

Comment:
```suggestion
```
Seems AD will only be visible in observability nav group

Reply:
addressed